web/templates/settings/settings_save_discard_widget.hbs

```diff
@@ -1,7 +1,7 @@
 {{#unless show_only_indicator}}
 <div class="save-button-controls hide">
     <div class="inline-block subsection-changes-save">
-        <button class="save-discard-widget-button button primary save-button" data-status="save">
+        <button class="save-discard-widget-button button primary save-button" data-status="save" id="save-discard-widget-id">
```

Comment:
Let's not add an extra ID when we can use the preexisting `.save-button` class for the same purpose,